api/provider/aws/test_aws/environment_create_test.go

```diff
@@ -261,30 +260,6 @@ func TestEnvironmentCreateDefaults(t *testing.T) {
 		AuthorizeSecurityGroupIngress(gomock.Any()).
 		Return(&ec2.AuthorizeSecurityGroupIngressOutput{}, nil)
 
-	// ensure we pass the default instance type, ami id, and user data to the launch configuration
```

Comment:
ah! no! This behavior hasn't been removed so we should not remove/reduce this test. Instead, we should add a new test to test the new behavior.

Reply:
Ah, I had mistakenly thought that this was duplicated in an earlier test; this functionality should be moved to a different test then, since the new default is to use a dynamic (Fargate) environment and that doesn't create LauncConfigurations or AutoScalingGroups.